gensrc/thrift/PlanNodes.thrift

```diff
@@ -114,6 +114,8 @@ struct TBrokerRangeDesc {
     7: optional Types.TUniqueId load_id
     // total size of the file
     8: optional i64 file_size
+    // columns parsed from file path
+    9: optional list<string> columns_from_path
```

Comment:
It's better to record the slot offset for `columns_from_path` or name it `num_of_columns_from_file`. Because we may add other `columns_from_xxx`, we should make it definite.
And you should comment that `columns_from_path` is after the columns read from file.